pkg/network/encoding/marshal/usm.go

```diff
@@ -48,6 +52,9 @@ func GroupByConnection[K comparable, V any](protocol string, data map[K]V, keyGe
 	byConnection := &USMConnectionIndex[K, V]{
 		protocol: protocol,
 		lookupFn: USMLookup[K, V],
+
+		// Experimental: Connection Rollups
+		enableConnectionRollup: config.SystemProbe.GetBool("service_monitoring_config.enable_connection_rollup"),
```

Comment:
can't we pass the value? we usually not reading the value directly in the internal code

Reply:
We don't have a `network.Config` instance available at the marshaler and I was just following the same "pattern" that I saw in other files from this package (like [here](https://github.com/DataDog/datadog-agent/blob/pedro%2Fconnection-rollup/pkg/network/encoding/marshal/dns.go#L34-L35), for example).

I think it's probably worth it figuring out a better pattern, but would leave this for a separate PR and change it for the whole package.

Reply:
Please open a jira ticket so it won't get lost